src/components/view/sources/sources.ts

```diff
@@ -0,0 +1,21 @@
+import './sources.css';
+import { TextSources } from '../../options';
+class Sources {
+  draw(data: TextSources[]) {
+    const fragment = document.createDocumentFragment();
+    const sourceItemTemp = document.querySelector('#sourceItemTemp');
```

Comment:
тогда ниже не придется писать `as HTMLTemplateElement`
```suggestion
    const sourceItemTemp: HTMLTemplateElement = document.querySelector('#sourceItemTemp');
```